.github/ISSUE_TEMPLATE.md

```diff
@@ -3,7 +3,10 @@
 _Delete this section if this is a feature request._
 
 * Skate version: _x.x.x_
+  <!--(run `npm list skatejs` from a terminal/cmd prompt and paste output below): -->
```

Comment:
This isn't actually displayed. Should probably use italics?

Reply:
nope we as maintainers don't wanna get disturbed by reading that :) it's just helper info for the issuer